package.json

```diff
@@ -163,5 +163,8 @@
   },
   "homepage": "https://google.github.io/zx/",
   "author": "Anton Medvedev <anton@medv.io>",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "volta": {
+    "node": "22.12.0"
```

Comment:
can we use just `22` here?

Reply:
No 

I will be uploading an issue soon.


<img width="888" alt="Screenshot 2024-12-20 at 18 01 47" src="https://github.com/user-attachments/assets/cd79abbc-43e7-47f5-bf6e-31ff5f70fc56" />